src/storage-client/src/controller/rehydration.rs

```diff
@@ -139,6 +140,8 @@ struct RehydrationTask<T> {
     sinks: BTreeMap<GlobalId, CreateSinkCommand<T>>,
     /// The upper frontier information received.
     uppers: BTreeMap<GlobalId, Antichain<T>>,
+    /// The since frontiers that have been observed.
```

Comment:
Might be worthwhile to highlight that uppers come back from the cluster, while sinces "go out" from the controller.